src/iidm/ThreeWindingsTransformer.cpp

```diff
@@ -129,11 +141,11 @@ double ThreeWindingsTransformer::Leg::getX() const {
 }
 
 bool ThreeWindingsTransformer::Leg::hasPhaseTapChanger() const {
-    return static_cast<bool>(m_phaseTapChanger);
+    return m_phaseTapChanger != nullptr;
 }
 
 bool ThreeWindingsTransformer::Leg::hasRatioTapChanger() const {
-    return static_cast<bool>(m_ratioTapChanger);
+    return m_ratioTapChanger != nullptr;
 }
```

Comment:
Not really efficient: `nullptr` will be implicitly converted to a `unique_ptr` whereas a `static_cast<bool>()` seems better.